app/templates/courses/project-gallery-view.jade

```diff
@@ -0,0 +1,24 @@
+flat-layout
+  .container.m-t-3
+    p
+      a(v-if="amTeacher", :href="backToClassroomUrl", data-i18n="courses.back_classroom")
+      a(v-else, href="/students", data-i18n="courses.back_courses")
+    div.text-center.m-t-2
+      h2(v-if="classroom")
+        {{ classroom.name }}
+      h1(v-if="course")
+        {{ courseName }}: {{ levelName }}
+      div(v-if="_.isEmpty(levelSessions)")
+        {{ $t('project_gallery.no_projects_published') }}
+      ul.project-list.m-t-3(v-else)
+        .row
+          li.col-xs-8.col-xs-offset-2.project-list-item(v-for="session in levelSessions")
+            span.creator-name(v-if="users && creatorOfSession(session)")
+              | {{ creatorOfSession(session).firstName }}
+              | {{ creatorOfSession(session).lastName }}.
+              | ({{ creatorOfSession(session).name }})
```

Comment:
Oh, there's space already between these three lines? Pipes do that? Did not know that!